micro_dl/input/tile_stack.py

```diff
@@ -18,7 +18,8 @@ def __init__(self,
                  tile_dict,
                  tile_size=[256, 256],
                  step_size=[64, 64],
-                 depth=1,
+                 depths=1,
+                 mask_depth=1,
```

Comment:
wondering if we need this variable here. Is there a case where mask_depth would be different than the ones in self.channel_depth?

Reply:
I think you would still need to know with channel it is matching.
E.g. in the preprocessing config depths are [3, 1] for the two channels to be tiled.
I could set the mask depth to be the max of depths, but are we sure that that will always be the case?